### ISSUE TITLE
Stop using `airflow celery stop` command

### ISSUE THREAD

**Issue by rafidka** [aws/amazon-mwaa-docker-images#105]
## Overview

TL;DR - The airflow celery stop command doesn't work when the worker is not running in daemon mode, which is the case for us.

The [implementation](https://github.com/apache/airflow/blob/2.9.2/airflow/providers/celery/cli/celery_command.py#L246-L261) of the celery stop command relies on the existence of the `airflow-worker.pid` file; if it doesn't exist, you end up getting [None here](https://github.com/apache/airflow/blob/2.9.2/airflow/providers/celery/cli/celery_command.py#L256-L258), and nothing happens. Now, the `airflow-worker.pid` file is only [generated if we pass the `--daemon` option](https://github.com/apache/airflow/blob/2.9.2/airflow/cli/commands/daemon_utils.py#L50-L57) when we call `airflow celery worker`; see help snapshot below. Since we are not using daemon mode [when we create the worker](https://github.com/aws/amazon-mwaa-docker-images/blob/main/images/airflow/2.9.2/python/mwaa/entrypoint.py#L558), no `airflow-worker.pid` file ends up being created, and hence calls to airflow celery stop ends up doing nothing. I have verified this by logging into the Docker container itself and executing `airflow celery stop` manually, and it didn't do nothing.

## Acceptance Criteria

- Avoid all reliance on `airflow celery stop` calls to stop Celery workers. We should instead just interrupt the worker ourselves.
- Potentially, the above is a bug in Airflow Celery Provider, so we should also create an issue to their repository to get it fixed.

## Additional Info

![image](https://github.com/aws/amazon-mwaa-docker-images/assets/442447/c5605d07-c149-48ec-9c78-88bd77f6060b)
